<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify_Class</class>
 <widget class="QMainWindow" name="Modify_Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Name_label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Name_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Cost_label">
       <property name="text">
        <string>Cost</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="Cost_spinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Start_time_label">
       <property name="text">
        <string>Start Time</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTimeEdit" name="Start_time_timeEdit">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Day">
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="Day_comboBox">
       <item>
        <property name="text">
         <string>Sunday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tuesday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wednesday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thursday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Friday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Saturday</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Location_label">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="Cap">
       <property name="text">
        <string>Cap</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="Cap_spinBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="Clothing_label">
       <property name="text">
        <string>Clothing</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QTextEdit" name="Clothing_textEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>95</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>693</width>
         <height>95</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="Description">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QTextEdit" name="Description_textEdit">
       <property name="maximumSize">
        <size>
         <width>680</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="Start_date_label">
       <property name="text">
        <string>Start Date</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDateEdit" name="Start_date_dateEdit"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="End_date">
       <property name="text">
        <string>End Date</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDateEdit" name="End_date_dateEdit"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="Age">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="Age_spinBox"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="Age_limit">
       <property name="text">
        <string>Age Limit</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QSpinBox" name="Age_limit_spinBox"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="Location_comboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="End_time_label">
       <property name="text">
        <string>End Time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="End_time_timeEdit">
       <property name="displayFormat">
        <string>HH:mm:ss </string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>*Required Fields                * 0 in Age or Age Limit represents no restriction (Leave 0's for All age classes) </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Submit_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back_btn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
